index.html

```diff
@@ -1143,15 +1143,6 @@ <h1>Projects Using Ace</h1>
                                 </li>
                             </ul>
                         </div>
-                        <div class="tab-pane fade" id="support">
```

Comment:
We could still have links to: https://github.com/ajaxorg/ace/discussions and https://github.com/ajaxorg/ace/issues

Reply:
Google groups are rather old and buggy, it is better if people use discussions on github.

Reply:
Do we have a link to Ace GitHub page on the website? I've only seen a link to `ace-builds`, which is a wrong place to submit issues.

Reply:
@whazor sorry, i had misread your comment, added Getting Help section to the main page. @InspiredGuy we had fork on githhub link at the top, but it is probably hard to notice.